mongodbatlas/resource_mongodbatlas_team.go

```diff
@@ -85,7 +85,14 @@ func resourceMongoDBAtlasTeamRead(d *schema.ResourceData, meta interface{}) erro
 	teamID := ids["id"]
 
 	team, _, err := conn.Teams.Get(context.Background(), orgID, teamID)
+
 	if err != nil {
+		// new resource missing
+		reset := strings.Contains(err.Error(), "404") && !d.IsNewResource()
+		if reset {
+			d.SetId("")
+			return nil
+		}
```

Comment:
We've had a few of these.   Should we perhaps do a ticket for an audit to check all resources or do as we've done, add as we find them?

Reply:
yea, agree with that :)

Reply:
https://jira.mongodb.org/browse/INTMDB-188